torch/export/exported_program.py

```diff
@@ -604,3 +604,8 @@ def get_shape_env(gm):
         if k not in shape_env.replacements
     }
     return range_constraints
+
+
+def combine_args_kwargs(args, kwargs):
```

Comment:
do we need this function still? I think we should inline this like

```
(arg, kwargs or {})
```

Reply:
It seems to be used in a lot of places 😅 https://www.internalfb.com/code/search?q=repo%3Afbcode%20combine_args_kwargs